src/coreclr/hosts/unixcorerun/.gitmirrorall

```diff
@@ -1 +0,0 @@
-This folder will be mirrored by the Git-TFS Mirror recursively.
```

Comment:
Sure, I'll keep gitmirrorall. I changed it to .gitmirror since we no longer have any subdirectories, but the behavior will be the same.